src/System.ComponentModel.TypeConverter/ref/project.json

```diff
@@ -8,9 +8,9 @@
     "System.Runtime": "4.0.0"
```

Comment:
We should probably update these package versions to use  the latest prerelease instead so that we are all compatible and have the right dependencies across ns1.7